Changelog.md

```diff
@@ -1,8 +1,12 @@
 # Changelog: devise-secure_password
 
-## 2.0.2 / 2021-12-08
+## 2.1.0 / 2021-12-08
```

Comment:
@geeksam quick date update here `2.1.0 / 2024-06-12`

Reply:
oops, made the change but forgot to commit and push it 🤦